antigen.zsh

```diff
@@ -737,20 +737,55 @@ antigen () {
 
 # Setup antigen's autocompletion
 _antigen () {
-    compadd        \
-        bundle     \
-        bundles    \
-        update     \
-        revert     \
-        list       \
-        cleanup    \
-        use        \
-        selfupdate \
-        theme      \
-        apply      \
-        snapshot   \
-        restore    \
-        help
+  local -a _1st_arguments
+  _1st_arguments=(
+    'bundle:Install and load the given plugin'
+    'bundles:Bulk define bundles'
+    'update:Update all bundles'
+    'revert:Revert the state of all bundles to how they were before the last antigen update'
+    'list:List out the currently loaded bundles'
+    'cleanup:Clean up the clones of repos which are not used by any bundles currently loaded'
+    'use:Load any (supported) zsh pre-packaged framework'
+    'theme:Switch the prompt theme'
+    'apply:Load all bundle completions'
+    'snapshot:Create a snapshot of all the active clones'
+    'restore:Restore the bundles state as specified in the snapshot'
+    'selfupdate:Update antigen itself'
+    'help:Print help message'
+  )
+
+  __bundle() {
+    _arguments \
+      '--loc=[Path to the location <path-to/location>]' \
```

Comment:
Bundle is missing 'type' (--btype) argument; which values can be either "theme" or "plugin". See [here](https://github.com/zsh-users/antigen/blob/master/antigen.zsh#L29) and here [here](https://github.com/zsh-users/antigen/blob/master/antigen.zsh#L62).